pkg/runctl/run_manager.go

```diff
@@ -736,3 +797,12 @@ func tektonStringParam(name string, value string) tekton.Param {
 		},
 	}
 }
+
+func toValidURL(indexURL string) (string, error) {
```

Comment:
```suggestion
func ensureValidURL(indexURL string) (string, error) {
```
`toValidURL` sounds like invalid URLs are getting fixed by this function